pkg/api/installopenshiftversions.go

```diff
@@ -0,0 +1,7 @@
+package api
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+// InstallOpenShiftVersions represents a List of OpenShift installable versions.
```

Comment:
Need to be consistent with the naming, now that we changed it: `InstallOpenShiftVersions` -> `InstallVersions`.